### ISSUE TITLE
Add all new icons

### ISSUE THREAD

**Issue by florianbouvot** [zaydek/heroicons.dev#28]
@codex-zaydek thank you for your great tools 👍  

I wonder if you can add all new icons : https://github.com/refactoringui/heroicons/commit/60e6750e1cee2477e62f62f54f4023690de7336c ?

Florian.

**Comment by zaydek** [comment]
Hey @florianbouvot! Sorry, just seeing this now.

Yes, I actually attempted to do this but the icons appear to be mis-exported: https://github.com/refactoringui/heroicons/issues/55. Steve (Schoger) acknowledged the problem here https://twitter.com/username_ZAYDEK/status/1265932196604305411 but I don’t think he’s had time to look into it yet.

I couldn’t find a way to ship all of the new icons properly when I looked at the code. If you think it _is_ possible as is just let me know.
